reports/models.py

```diff
@@ -190,6 +199,20 @@ class SectionParagraph(Section):
         )
     )
 
+    class Meta:
+        verbose_name = _("Paragraphe libre")
+
+    def get_context(self, context):
+        env = SandboxedEnvironment()
+        request_data = PermitRequestPrintSerializer(context["permit_request"]).data
+
+        wot = context["work_object_type"]
+        wot_key = (
+            f"{wot.works_object.name} ({wot.works_type.name})"  # defined by serializer
+        )
+        request_props = request_data["properties"]["request_properties"][wot_key]
+        amend_props = request_data["properties"]["amend_properties"][wot_key]
```

Comment:
obsolete (probably wrong merge commit resolution). This is now done in prepare_context and _render_user_template just below)